applications/nice1-tools-ecosystem.md

```diff
@@ -0,0 +1,278 @@
+
+
+### Team Org Repos
+
+- https://github.com/nice1-blockchain 
+- https://github.com/nice1-blockchain/nice1.link
+- https://github.com/nice1-blockchain/nice-react-tools
+- https://github.com/nice1-blockchain/nice1-plugin
+- https://github.com/nice1-blockchain/nice1-plugin-unity
+
+
+
+
+### Team Member Repos
+
+- https://github.com/alberuiza 
+- https://github.com/elboletaire 
+
+### Team LinkedIn Profiles
+
+- https://www.linkedin.com/in/javignice1foundation/ 
+- https://www.linkedin.com/in/antoniollopezfernandez/
+- https://www.linkedin.com/in/jos%C3%A9-mar%C3%ADa-mora-92a1abb5/ 
+- https://www.linkedin.com/in/elan-f-hernando-general-fyah-16710b1a4/
+- https://www.linkedin.com/in/albertoruizaguilar/ 
+- https://www.linkedin.com/in/ocasajuana/ 
+- https://www.linkedin.com/in/javiersacristan/
+- https://www.linkedin.com/in/raquel-guirado/ 
+- https://www.linkedin.com/in/ritosalgado/
+
+
+## Development Status
+
+At Nice1 we have been working for several months on the development of our tools. The current status of the developments:
+
+- A. Nice1 Plugins:
+
+  - A1. IDEs:
+A1.1 Unity: We have developed the first version of the plugin, available at the following urls:
+https://github.com/nice1-blockchain/nice1-plugin-unity
+https://docs.nice1.dev/
+
+  - A2. Functionalities
+
+    - A2.2 LegendaryLegends Demo: A PC demo is available to test the Nice1 licensing system: Legendary Legends. 
+https://github.com/nice1-blockchain/LegendaryLegends
+https://twitter.com/Nice1BlockChain/status/1456053844367089665?s=20&t=y56kEGW1ClGQ7fIfiB7JzA
+
+    - A2.3 Extend EOS networks support: Work in progress. We want to share this functionality in the next version. 
+
+- B. Nice1 Link: The login functionality, contract connection to add a nickname, and the news about Nice1 have been implemented. You can test it here: https://nice1-blockchain.gitlab.io/nice1-link/
+
+*We were contacted by Nathan to schedule a meeting with him and our development/operations team, in order to let him know about our project and in which we explained our objectives and means to reach them, as well as to talk about our experience in building on EOS.*
+
+
+
+
+## Development Roadmap
+
+### Overview
+Total Estimated Duration: 6 months
+
+Full-Time Equivalent (FTE): Two (2)
+
+Total Costs: $49,000 USD
+
+Note: Some of the milestones can possibly overlap and be done in parallel.
+
+## Milestone Summary
+
+
+### Milestone 1 - Nice1 Plugin Unity
+- Estimated duration: 2 months
+- FTE: 1
+- Costs: $8.300 USD
+
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| 0a. | License |  Apache 2.0  |
+| 0b. | Documentation | Documentation for Unity developers. |
+| 0c. | Unit tests | Unit tests for the plugin. |
+| 1. | Webview | Develop a webview plugin to call anchor-link functions. |  
+| 2. | C# Anchor-Link | Analyze the problem to develop a c# Anchor-link SDK. |  
+| 3. | Platform | Mac, Android and iOS version.|  
```

Comment:
Since this is a Unity plugin (the same question is for the Unreal plugin), you shouldn't be limited to MacOS, Android and iOS correct? What about Linux and Windows 10/11?

Reply:
Yes, of course, we are going to develop it for Windows and Linux in both tools, Unity and Unreal.